2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
I considered adding a formatType so that we could specify 'thousands' for currency, but then I couldn't justify adding a formatType to all of the other items. In the end I just went a comma-separated valuesfor the currency format since in the code we could match specifically on the value (e.g. `if (format === 'currency,thousands')`). And if we ever needed too, it would be trivial to split the format value on the comma so that each format was an array of values where the first value was the base format value (string, number, date) and the 2nd value was a type specific format.